examples/oneapi/dpc/makefile_lnx

```diff
@@ -95,7 +95,8 @@ COPTS := -std=c++17 \
          -Werror \
          -Wno-unused-parameter \
          -I./source \
-         -I"$(DAALROOT)/include"
+         -I"$(DAALROOT)/include" \
+         -fsycl-device-code-split=per_kernel
```

Comment:
Is it needed in case of new interfaces and static linking?

Reply:
```suggestion
```